packages/api-react/src/services/wallet.ts

```diff
@@ -79,7 +79,7 @@ export const walletApi = apiWithTag.injectEndpoints({
               wallets.map(async (wallet: Wallet) => {
                 const { type } = wallet;
                 const meta: any = {};
-                if (type === WalletType.CAT) {
+                if ([WalletType.CAT, WalletType.CRCAT].includes(type)) {
```

Comment:
Since this appears to be repeated in quite a few places, do you think it would make sense to write a small function to encapsulate this logic?